metadata.json

```diff
@@ -104,22 +96,42 @@
       "name": "simp/pupmod",
       "version_requirement": ">= 7.1.0 < 8.0.0"
     },
+    {
+      "name": "simp/resolv",
+      "version_requirement": ">= 0.1.1 < 1.0.0"
+    },
     {
       "name": "simp/rsync",
       "version_requirement": ">= 6.0.0 < 7.0.0"
     },
     {
-      "name": "simp/rsyslog",
-      "version_requirement": ">= 7.0.0 < 8.0.0"
+      "name": "simp/selinux",
+      "version_requirement": ">= 2.3.0 < 3.0.0"
     },
     {
-      "name": "simp/simpcat",
+      "name": "simp/simp_openldap",
       "version_requirement": ">= 6.0.0 < 7.0.0"
     },
+    {
+      "name": "simp/simp_rsyslog",
+      "version_requirement": ">= 0.3.0 < 1.0.0"
+    },
     {
       "name": "simp/simplib",
       "version_requirement": ">= 3.12.0 < 4.0.0"
     },
+    {
+      "name": "simp/ssh",
+      "version_requirement": ">= 6.5.1 < 7.0.0"
+    },
+    {
```

Comment:
Missing simp/sudo ?  Used by numerous classes (e.g., simp::sudoers, simp::sudoers::alias).  simp::sudoers is included by simp::admin, which, in turn is in the class list in data-in-modules.